client/my-sites/stats/stats-notices/index.tsx

```diff
@@ -33,9 +34,13 @@ const NewStatsNotices = ( { siteId, isOdysseyStats }: StatsNoticesProps ) => {
 		hasLoadedPurchases;
 
 	const showFreePlanPurchaseSuccessNotice = config.isEnabled( 'stats/paid-stats' ) && false;
+	const showPaidPlanPurchaseSuccessNotice = config.isEnabled( 'stats/paid-stats' ) && true;
```

Comment:
According to the testing instructions, the original second operand should be `false`.

```suggestion
	const showPaidPlanPurchaseSuccessNotice = config.isEnabled( 'stats/paid-stats' ) && false;
```

Reply:
Ah, you're right. Thanks!